project/src/components/app/app.jsx

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
+import MainPage from '../main-page/main-page';
+import PropTypes from 'prop-types';
```

Comment:
```suggestion
import PropTypes from 'prop-types';

import MainPage from '../main-page/main-page';
```

Не мешай импорты в кучу. 
Для удобства, лучше импортировать сначала библиотеки, а потом кастомные вещи.